src/amo/sagas/index.js

```diff
@@ -12,10 +12,10 @@ import addons from 'core/sagas/addons';
 
 // Export all sagas for this app so runSaga can consume them.
 export default function* rootSaga() {
-  yield [
+  yield all([
```

Comment:
No need, I believe you! I see in the release notes that it's no different than before; it's just more explicit. Works for me.